srcpkgs/intel-opencl-clang/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'intel-opencl-clang'
+pkgname=intel-opencl-clang
+version=10.0.0.1
+revision=1
+_version=10.0.0-1
```

Comment:
`version` should be used, rather that be purely decorative. please use `${version%.*}-${version##*.}` here.

Reply:
done